src/web/Content/client/dumpling.py

```diff
@@ -944,6 +956,24 @@ def _add_key_if_not_exists(dictProp, key, val):
         if not key in dictProp:
             dictProp[key] = val
 
+    @staticmethod
+    def _create_hang_dump(pid, outpath, debuggerpath):
+        osStr = platform.system().lower()
+        command = ""
+        if osStr == 'linux':
+            command = "./" + debuggerpath + " " + pid + " --name " + outpath + "\memdum" + pid + ".dmp"
+        elif osStr == 'windows':
+            outputpath = outpath + "\memdum" + pid + ".dmp"        
+            command = debuggerpath + " -p "+ pid + " -c " + '".dump /ma '+outputpath+';.detach;q"'
+        else:
+            return
+
+        print "creating dump"
```

Comment:
does the command get logged? might be worth doing (especially if there's an error)

Reply:
as much as possible every possible problem should be diagnosable from reading the log.